cluster/src/main/java/org/apache/iotdb/cluster/utils/nodetool/function/NodeToolCmd.java

```diff
@@ -72,7 +83,9 @@ private MBeanServerConnection connect() {
     try {
       String jmxURL = String.format(JMX_URL_FORMAT, host, port);
       JMXServiceURL serviceURL = new JMXServiceURL(jmxURL);
-      JMXConnector connector = JMXConnectorFactory.connect(serviceURL);
+      Map<String, Object> environment = new HashMap<>(1);
```

Comment:
`Collections.singletonMap` may be enough in this case

Reply:
Thanks for your remind, i'll change it